### PR TITLE
Gcp deploy

### PR DESCRIPTION
**Pull request by bkochauri-memphis**

### CODE REVIEW

**Comment by idanasulinmemphis** [comment]
@valeraBr is it ok with you?

File: Jenkinsfile [hunk 1]
```diff
@@ -17,13 +20,18 @@ pipeline {
             }
         }
     }
-
+	  
+	
     stage('Create memphis namespace in Kubernetes'){
         steps {
             sh """
+            sudo yum install lsof -y
```
Comment (valeraBr) [inline Jenkinsfile:28]: Please add it to image @bkochauri-memphis

Comment (bkochauri-memphis) [reply Jenkinsfile]: @valeraBr  Its done.

**Review by valeraBr — APPROVED** [review]

**Comment by bkochauri-memphis** [comment]
@idanasulinmemphis  please approve when you get time.

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**